Cargo.toml

```diff
@@ -53,6 +54,7 @@ send_wrapper = "0.6"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.61"
 rayon = "1.0.2"
+rust_decimal = { version = "1.18.0", features = ["std"] }
```

Comment:
Is this the oldest version which supports this functionality of our PR? If not, I would be glad if we could require the oldest version that is semver-compatible with 1.18.0.

Reply:
Honestly I don't even remember why 1.18.0 at this point. I know originally we really wanted to use `from_str_exact()` but that didn't come in until 1.21.0 which had a MSRV of 1.54 which is newer than what pyo3 has currently but is what pyo3 wants to move to. 1.19.0 and newer wants a newer version of arrayvec which wants const generics from 1.51 as well.

At this point all the call sites I see are using the more basic calls that I believe are there for 1.0 so I'll just go back to that. I originally had that in #2774 but it wasn't really compatible.